ziro-export-unity.ps1

```diff
@@ -43,10 +42,49 @@ function Execute-GetOnUnity {
         $Resources = $Response.$ResourceName
         $ResourcesArray += $Resources
     }
-
+    
     $JsonOutput = ConvertTo-Json $ResourcesArray
-    $JsonOutput | Out-File -FilePath $OutputFilePath
+
+    if ($OutputFileName) {
+        $OutputFilePath = "output-unity/" + $OutputFileName
+        $JsonOutput | Out-File -FilePath $OutputFilePath
+    }
+    
     return $JsonOutput | ConvertFrom-Json
+
+
+}
+
+function Export-Greetings {
+    param (
+        $Greetings,
+        [string]$CallHandlerId,
+        [string]$FolderName
+    ) 
+    $ProgressCount = 0
+    foreach ($Greeting in $Greetings) {
+        $ProgressCount++
+
+        $PlayWhat = [int]$Greeting.PlayWhat
+        $Enabled = [System.Convert]::ToBoolean($Greeting.Enabled)
+
+        if ($PlayWhat -eq 1 -and $Enabled -eq $true) {
+            $GreetingStreamFiles = Invoke-GetOnUnity $UnityHost ('/vmrest/handlers/callhandlers/' + $CallHandler.ObjectId + "/greetings/" + $Greeting.GreetingType + "/greetingstreamfiles") $Credential $null 'GreetingStreamFile'
```

Comment:
Should this be using `$CallHandlerId` instead of `$CallHandler.ObjectId`?

Reply:
Indeed